### PR TITLE
WRN-1381: Migrate Input and InputField unit tests to Testing Library.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated tests from Enzyme to Testing Library.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Changed test structure to work with Testing Library.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-1381

### CODE REVIEW

File: Input/tests/InputField-specs.js [hunk 1]
```diff
@@ -1,58 +1,68 @@
...
+			<InputField data-testid="inputField" />
 		);
 
-		expect(subject.find('input')).toHaveLength(1);
+		const inputField = getByTestId('inputField');
+		const expected = 'inputField';
+		const actual = inputField.className;
+
+		expect(actual).toContain(expected);
 	});
```
Comment (adrian-cocoara-lgp) [inline Input/tests/InputField-specs.js:20]: You can simply check if input is in the document, and not look for className. What do you think?
```suggestion
	test('should have an input element', () => {
		const {getByTestId} = render(
			<InputField data-testid="inputField" />
		);

		const inputField = getByTestId('inputField');

		expect(inputField).toBeInTheDocument();
	});
	
```

Comment (alexandrumorariu) [reply Input/tests/InputField-specs.js:18]: Done

**Review by adrian-cocoara-lgp — APPROVED** [review]
GJ!

**Review by daniel-stoian-lgp — COMMENTED** [review]
A few comments :)

File: Input/tests/Input-specs.js [hunk 2]
```diff
@@ -1,393 +1,377 @@
...
+		const {getAllByRole} = render(
 			<FloatingLayerController>
 				<Input open />
 			</FloatingLayerController>
 		);
 
-		const expected = true;
-		const actual = subject.find('FloatingLayer').prop('open');
+		const expected = 2;
+		const actual = getAllByRole('button').length;
```
[the remark this answers is not in the document]
Comment (alexandrumorariu) [reply Input/tests/Input-specs.js:18]: I will check if there is anything in the floating layer.

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Pull request merged**